TESTING.asciidoc

```diff
@@ -313,39 +321,28 @@ a bit more performance out of the process:
 vagrant plugin install vagrant-cachier
 --------------------------------------
 +
-. Validate your installed dependencies:
+
+. You can run all of the OS packaging tests with `./gradlew distroTest`. This
```

Comment:
This isn't quite true. `distroTest` is all of the newer java based tests, but does not include the legacy bats tests. To run both, the `packagingTest` task must be used.

Reply:
Ah, I didn't know that. Thank you for the correction.